### ISSUE TITLE
Provide alt text for avatar image

### ISSUE THREAD

**Issue by VYSY0** [VYSY0/vysy0.github.io#3]
The <img> tag in the main header lacks an alt attribute. Add descriptive alt text for accessibility.
